cmsranking/static/TimeView.js

```diff
@@ -109,7 +119,7 @@ var TimeView = new function () {
             $("#TimeView").addClass("current post_cont");
             full_time = true;
         } else {
-            if (cur_time < c['begin']) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'server_time' is not in camel case.](https://www.codacy.com/app/s-maggiolo/cms/file/2192640431/issues/source?bid=3167167&fileBranchId=3167181#l122)